Makefile

```diff
@@ -0,0 +1,23 @@
+# Variables
+ManyWellCMD=python examples/many_well.py -m training.seed=0,1,2 # by default seeds are run in parellel
```

Comment:
@VincentStimper you can use this to run all of the experiments for the many well problem. If you don't want to run all seeds in parellel in a single command then you can edit this line to instead be `training.seed=0`